lib/externalGrader.sql

```diff
@@ -10,7 +10,7 @@ FROM
   LEFT JOIN submissions AS s ON (s.id = gj.submission_id)
   LEFT JOIN variants AS v ON (v.id = s.variant_id)
   LEFT JOIN questions AS q ON (q.id = v.question_id)
-  LEFT JOIN pl_courses AS c ON (c.id = q.course_id)
+  LEFT JOIN pl_courses AS c ON (c.id = v.course_id)
```

Comment:
LGTM. I followed the course out of this query, and it's used to copy over `serverFilesCourse` into the grading job, which is what we want.